client/lib/user-agent-utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function currentUserAgent() {
+	return navigator.userAgent;
+}
```

Comment:
The reasoning behind this was that I'd expand this class to include other tests outside of just `isChromeOS` as I noticed in a few other spots in Calypso we do some checking against the userAgent.